### PR TITLE
Add Chinese readme

### PR DESCRIPTION
**Pull request by coryma**
Add Chinese readme

### CODE REVIEW

**Comment by mrutkows** [comment]
I have asked Daisy Guo on our CRL team to review this PR and its contents. @daisy-ycguo

File: README-zhTW.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
+# 於 IBM Cloud Functions 上使用無服務器 (Serverless) APIs  (運用 Apache OpenWhisk)
+
+*Read this in other languages: [English](README.md).*
+
+該專案主要用來了解無服務器、事件驅動式架構如何執行能動態擴展以符合資源需求的程式，用以回應HTTP REST API的呼叫。 程式在調用 API 端點之前不會消耗任何資源。 調用它們時，會調配資源以獨立地配置每個HTTP請求所需的當下資源負載。
+
+專案中包含四個 JavaScript 編寫的動作 （運行在 IBM Cloud Functions 使用 Apache OpenWhisk）操作（用JavaScript編寫），四個動作用在 MySQL 資料庫中寫入和讀取資料。 示範每項動作是如何讓資料服務及執行邏輯一起運作以回應HTTP請求。
```
Comment (daisy-ycguo) [inline README-zhTW.md]: 專案中包含四個 運行在 IBM Cloud Functions  （使用 Apache OpenWhisk）的動作（用JavaScript編寫），用在 MySQL 資料庫中寫入和讀取資料。

File: README-zhTW.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+第一個動作對應到HTTP POST請求，將貓的名字跟顏色參數新增至資料庫中。 第二個動作會對應至PUT請求以更新現有貓資料的欄位值。 第三個動作會對應至傳回指定貓的資料的GET請求。 第四個動作會刪除指定的貓的資料。
+
+IBM Cloud上 的 Node.js 執行系統 (runtime) 會提供內建的npm模組白名單。 此範例也會示範如何使用自訂動作將其它 Node.js 相依模組（例如MySQL用戶端npm）包裝在ZIP檔中以提供進一步的整合。
```
Comment (daisy-ycguo) [inline README-zhTW.md]: 此範例也會示範如何將其它 Node.js 相依模組（例如MySQL用戶端npm）和自訂動作打包到ZIP檔中以提供更大的擴展性。

File: README-zhTW.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+2. [建立 OpenWhisk 動作及對映](#2-create-openwhisk-actions-and-mappings)
+3. [測試 API 端點](#3-test-api-endpoints)
+4. [刪除 動作 及 對映](#4-delete-actions-and-mappings)
+5. [使用手動部署重新建立](#5-recreate-deployment-manually)
+
+## 1. 準備 MySQL
+
+登入 IBM Cloud 並且建立一個 MySQL 環境 [Compose for MySQL](https://console.ng.bluemix.net/catalog/services/compose-for-mysql/). 
+
+進入 [Compose](https://console.ng.bluemix.net/catalog/services/compose-for-mysql/), 於畫面右方選單, 點選 `服務認證`, 檢視內容
```
Comment (daisy-ycguo) [inline README-zhTW.md]: 这里和原文README.md不一致：

File: README-zhTW.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+bx wsk action delete cat/cat-put
+bx wsk action delete cat/cat-get
+bx wsk action delete cat/cat-delete
+bx wsk package delete cat
+```
+
+## 疑難排除
+
+首先在OpenWhisk活動日誌中檢查錯誤。 執行`bx wsk activation poll`指令可持續在畫面上顯示活動日誌，或使用 [點選 IBM Cloud Functions 上的監視選項] 讀取活動日誌詳細訊息 (https://console.ng.bluemix.net/openwhisk/dashboard).
+
```
Comment (daisy-ycguo) [inline README-zhTW.md]: 或使用 [點選 IBM Cloud Functions 上的監視選項] (https://console.ng.bluemix.net/openwhisk/dashboard)讀取活動日誌詳細訊息 .

File: README-zhTW.md (hunk 3 above)
Comment (coryma) [reply README-zhTW.md]: I found Bluemix doesn't support ClearDB anymore, so I didn't translate ClearDB part.

Comment (daisy-ycguo) [reply README-zhTW.md]: Fine. If you don't want to include ClearDB in this README, I think line 18 should also be changed, correspondingly.

**Comment by daisy-ycguo** [comment]
@coryma Let me know if you need help to change this PR and re-submit.

**Comment by daisy-ycguo** [comment]
@coryma Please remove the changes of .gitignore from your Pull Request. Then I'm ok to merge. Thanks.

**Pull request merged**